library/alloc/src/collections/vec_deque/into_iter.rs

```diff
@@ -17,7 +17,13 @@ pub struct IntoIter<
     T,
     #[unstable(feature = "allocator_api", issue = "32838")] A: Allocator = Global,
 > {
-    pub(crate) inner: VecDeque<T, A>,
+    inner: VecDeque<T, A>,
+}
+
+impl<T, A: Allocator> IntoIter<T, A> {
+    pub(super) fn from(inner: VecDeque<T, A>) -> Self {
```

Comment:
nit: I feel like I would name this function new, not from. It's not in  a From impl...

Reply:
Renamed the function.